python/src/ops.cpp

```diff
@@ -2931,19 +2929,46 @@ void init_ops(py::module_& m) {
       "load",
       &mlx_load_helper,
       "file"_a,
+      "format"_a = none,
       py::pos_only(),
```

Comment:
This is a very nit-picky comment in an amazing PR but I ask your understanding :-) .

I would move the format to be optionally a keyword argument ie after `py::pos_only()`. I imagine it is very readable to do `mx.load(my_python_stream, format="safetensors")`.